packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -107,6 +129,53 @@ void main() {
     expect(tester.testTextInput.setClientArgs!['inputAction'], equals(serializedActionName));
   }
 
+  // Related issue: https://github.com/flutter/flutter/issues/98115
+  testWidgets('test editable text scheduleShowCaretOnScreen with no animation when the window changes its metrics',
+          (WidgetTester tester)
+      async {
+
+    final ScrollController scrollController = ScrollController();
+    final Widget widget = MaterialApp(
+      home: Scaffold(
+        body: SingleChildScrollView(
+          controller: scrollController,
+          child: Column(
+            children: <Widget>[
+              Column(
+                children: List<Widget>.generate(
+                    5,
+                        (_) => Container(
+                      height: 1200.0,
+                      color: Colors.black12,
+                    )),
```

Comment:
Nit: Can you format this with 2 spaces and no double parenthesis?  I think like this:

```suggestion
                children: List<Widget>.generate(
                  5,
                  (_) => Container(
                    height: 1200.0,
                    color: Colors.black12,
                  ),
                ),
```